dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -85,6 +91,44 @@ private PageResourceHelper() {
 
     }
 
+    public void saveContent(final String pageId, final List<PageContainerForm.ContainerEntry> containerEntries) throws DotDataException {
+        final List<MultiTree> multiTres = new ArrayList<>();
```

Comment:
Use ImmutableList

Reply:
it can't be immutable because I add items it a few lines after